lib/stat_tracker.rb

```diff
@@ -57,6 +57,29 @@ def count_of_teams
     @teams.count
   end
 
+
+
+
+
+
+  def count_of_games_by_season
+    @games.each_with_object(Hash.new(0)) { |game, game_count| game_count[game.season] += 1 }
+  end
+ 
+
+  def total_goals_by_season
+    total_goals_by_season = @games.each_with_object(Hash.new(0.0)) { |game, hash| hash[game.season] += game.away_goals + game.home_goals}
```

Comment:
I'm a big fan of .each_with_object as it creates and iterates all at once. Anytime there is a hash creation then .each_with_object is ideal..